std/special/compiler_rt/addXf3.zig

```diff
@@ -0,0 +1,191 @@
+// Ported from:
+//
+// https://github.com/llvm-mirror/compiler-rt/blob/92f7768ce940f6437b32ecc0985a1446cd040f7a/lib/builtins/fp_add_impl.inc
```

Comment:
I've referenced the git mirror here regarding #1504 since it was easier. Can switch to the svn-id however if there is a preference.

Reply:
There's an official git repository now: https://github.com/llvm/llvm-project